presto-main/src/test/java/com/facebook/presto/block/TestBlockBuilder.java

```diff
@@ -117,6 +118,9 @@ public void testNewBlockBuilderLike()
     @Test
     public void testNewBlockBuilderLikeForLargeBlockBuilder()
     {
+        if (true) {
```

Comment:
can you instead disable it in the `@Test` annotation like`@Test(enabled = false)`.  That's how we usually disable things

Reply:
Then it doesn't show up in the count as skipped?
Originally that is what I did, but then it hid the fact we were skipping a test.

Reply:
interesting. maybe we should be doing that in general then.